.storybook/data/tokens.json

```diff
@@ -353,21 +353,6 @@
   "eds-theme-color-button-icon-error-text": "#BD0044",
   "eds-theme-color-button-icon-error-text-hover": "#BD0044",
   "eds-theme-color-button-icon-error-text-active": "#FFFFFF",
-  "eds-theme-color-button-link-brand": "#5751D2",
-  "eds-theme-color-button-link-brand-hover": "#3E42B1",
-  "eds-theme-color-button-link-brand-active": "#3E42B1",
-  "eds-theme-color-button-link-neutral": "#5D6369",
-  "eds-theme-color-button-link-neutral-hover": "#383C43",
-  "eds-theme-color-button-link-neutral-active": "#383C43",
-  "eds-theme-color-button-link-success": "#007249",
-  "eds-theme-color-button-link-success-hover": "#005939",
-  "eds-theme-color-button-link-success-active": "#005939",
-  "eds-theme-color-button-link-warning": "#AC3400",
-  "eds-theme-color-button-link-warning-hover": "#842800",
-  "eds-theme-color-button-link-warning-active": "#842800",
-  "eds-theme-color-button-link-error": "#BD0044",
-  "eds-theme-color-button-link-error-hover": "#8F0134",
-  "eds-theme-color-button-link-error-active": "#8F0134",
```

Comment:
I originally added the link token colors because the colors being used weren't all in the second tier tokens. That's no longer the case, so I just removed the tokens entirely.